package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "eslint-config-foreground",
-  "version": "0.0.9",
+  "version": "1.0.0",
```

Comment:
How do you typically do this @Pedromdrp , a lot of dependencies needed to be updated for eslint 8, also added explicit node and npm versions here.

Reply:
I normally run `npm run release`.